<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Min latitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_latitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt; 
color: rgb(227, 227, 227);</string>
   </property>
   <property name="text">
    <string>Enter a range of coordinates to view a list of airports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submin_coordinates_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(227, 227, 227); font-size: 14pt;
background-color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Max latitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_latitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Min longitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_longitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Max longitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_longitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
